model/src/main/java/org/cloudfoundry/identity/uaa/zone/UserConfig.java

```diff
@@ -27,4 +27,14 @@ public List<String> getDefaultGroups() {
     public void setDefaultGroups(List<String> defaultGroups) {
         this.defaultGroups = defaultGroups;
     }
+
+    private List<String> allowedGroups = null;
+
```

Comment:
Combine the default and the allowed Groups
e.g. if you have entries in defaultGroups, they are automatically in allowed as far as I understood?

Reply:
done